style.css

```diff
@@ -0,0 +1,222 @@
+html {
+    box-sizing: border-box;
+}
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+body {
+    background-color: #ECEEF1;
+    font-family: AvenirNext,Avenir,"Helvetica Neue",Helvetica,Arial,sans-serif;
+}
+
+#main-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    width: 100vw;
+    height: 10vh;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    color: #505050;
+}
+
+#forgot-pwd {
+    margin-right: 0.5em;
+    text-align: right;
+    line-height: 1.3em;
+}
+
+#signin-link {
+    padding-right: 2em;
+    display: flex;
+    font-size: 12px;
+}
+
+#signin {
+    padding: 0.6em 2em;
+    background-color: #0077C5;
+    border: 1px solid #4C4C4C;
+    border-radius: 3px;
+    font-size: 12px;
+    color: white;
+}
+
+a {
+    text-decoration: none;
+    color: #0077C5;
+    font-size: 12px;
+}
+
+a:visted {
```

Comment:
Typo. I think you intended to write `visited`

Reply:
Updated spelling from visted to visited